From 9cbdc0472973c23b4e93200ed30afecbd0ef24d6 Mon Sep 17 00:00:00 2001
From: Matt Porter <mporter@konsulko.com>
Date: Tue, 27 Jun 2017 21:50:23 +0300
Subject: [PATCH] disable module-role-cork by default

module-role-cork's hardcoded default corking policies can conflict
with module-router and AudioManager's configurable routing policies.
Disable it to avoid unwanted corking changes on streams.

Signed-off-by: Matt Porter <mporter@konsulko.com>
---
 src/daemon/default.pa.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/daemon/default.pa.in b/src/daemon/default.pa.in
index d46bd5e..023e680 100755
--- a/src/daemon/default.pa.in
+++ b/src/daemon/default.pa.in
@@ -151,7 +151,7 @@ load-module module-systemd-login
 load-module module-position-event-sounds
 
 ### Cork music/video streams when a phone stream is active
-load-module module-role-cork
+#load-module module-role-cork
 
 ### Modules to allow autoloading of filters (such as echo cancellation)
 ### on demand. module-filter-heuristics tries to determine what filters
-- 
2.11.0

